runtime/dart_isolate.h

```diff
@@ -176,6 +176,12 @@ class DartIsolate : public UIDartState {
   /// @param[in]  io_manager                  The i/o manager.
   /// @param[in]  unref_queue                 The Skia unref queue.
   /// @param[in]  image_decoder               The image decoder.
+  /// @param[in]  image_generator_registry    Cascading registry of image
```

Comment:
Every time we add a new parameter to this my soul dies a little bit.

If you have any interest, it would be nice to resolve https://github.com/flutter/flutter/issues/71620 so we don't have to keep updating the 10 or however many files this ends up touching every time we do this :)

Reply:
Oh yeah, I agree this would be nice to make passing around all this shared state simpler. I'd prefer to do it in a simpler follow-up PR if you don't mind.

Reply:
Yeah, definitely a separate PR